diff --git a/dist/generate/dayjs.js b/dist/generate/dayjs.js
index 0f546abaabcfc6879a7b4cf95edfc5e399386eb5..0e3f810723a166577cb450869d5a904ea9fbd757 100644
--- a/dist/generate/dayjs.js
+++ b/dist/generate/dayjs.js
@@ -1,10 +1,10 @@
 import dayjs from "dayjs";
-import advancedFormat from "dayjs/plugin/advancedFormat";
-import customParseFormat from "dayjs/plugin/customParseFormat";
-import localeData from "dayjs/plugin/localeData";
-import weekday from "dayjs/plugin/weekday";
-import weekOfYear from "dayjs/plugin/weekOfYear";
-import weekYear from "dayjs/plugin/weekYear";
+import advancedFormat from "dayjs/plugin/advancedFormat.js";
+import customParseFormat from "dayjs/plugin/customParseFormat.js";
+import localeData from "dayjs/plugin/localeData.js";
+import weekday from "dayjs/plugin/weekday.js";
+import weekOfYear from "dayjs/plugin/weekOfYear.js";
+import weekYear from "dayjs/plugin/weekYear.js";
 dayjs.extend(customParseFormat);
 dayjs.extend(advancedFormat);
 dayjs.extend(weekday);
